community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/StoreVersionCheck.java

```diff
@@ -83,7 +83,7 @@ private String typeDescriptor( String expectedVersion )
         return expectedVersion.substring( 0, spaceIndex );
     }
 
-    private String readVersion( PagedFile pagedFile, String expectedVersion ) throws IOException
+    public static String readVersion( PagedFile pagedFile, String expectedVersion ) throws IOException
```

Comment:
Yes, but I do not know where this utility class would go, or what it would be named. it would really be a nice thing to have, for general store trailer handling through the page cache.